Makefile

```
@@ -20,16 +20,15 @@ ___CONTRACTS___: ##
 build-contracts: ## builds all contracts
 	cd contracts && forge build
 
-deploy-eigenlayer-contracts-to-anvil-and-save-state: ## Deploy eigenlayer
-	./tests/anvil/deploy-eigenlayer-save-anvil-state.sh
+deploy-eigenlayer: ## Deploy eigenlayer
+	./tests/anvil/deploy-eigenlayer.sh
 
-deploy-incredible-squaring-contracts-to-anvil-and-save-state: ## Deploy avs
-	./tests/anvil/deploy-avs-save-anvil-state.sh
+deploy-avs: ## Deploy avs
+	./tests/anvil/deploy-avs.sh
 
-deploy-all-to-anvil-and-save-state: deploy-eigenlayer-contracts-to-anvil-and-save-state deploy-incredible-squaring-contracts-to-anvil-and-save-state ## deploy eigenlayer, shared avs contracts, and inc-sq contracts 
-
-start-anvil-chain-with-el-and-avs-deployed: ## starts anvil from a saved state file (with el and avs contracts deployed)
-	./tests/anvil/start-anvil-chain-with-el-and-avs-deployed.sh
+deploy-el-and-avs-contracts:
+	$(MAKE) deploy-eigenlayer
+	$(MAKE) deploy-avs
```

We can express this in one single line:

```suggestion
deploy-el-and-avs-contracts: deploy-eigenlayer deploy-avs
```